docs/Manuals/Server/options.rst

```diff
@@ -0,0 +1,1176 @@
+  the distance is calculated as *Manhattan distance*, that is, the sum of the displacements along the
+  :math:`x` and :math:`y` directions.
+
+``tradeworldrelpct``
+  :strong:`Value (Min, Max)`: 50 (0, 100)
+
+  :strong:`Description`: How largely trade distance is relative to world size. When determining trade between
+  cities, the distance factor can be partly or fully relative to world size. This setting determines how big
+  percentage of the bonus calculation is relative to world size, and how much only absolute distance matters.
+
+``trading_city``
+  :strong:`Value (Min, Max)`: enabled
+
+  :strong:`Description`: City trading via treaty. If turned off, trading cities in the diplomacy dialog is not
+  allowed.
+
+``trading_gold``
+  :strong:`Value (Min, Max)`: enabled
+
+  :strong:`Description`: Gold trading via treaty. If turned off, trading gold in the diplomacy dialog is not
+  allowed.
+
+``trading_tech``
+  :strong:`Value (Min, Max)`: enabled
+
+  :strong:`Description`: Technology trading via treaty. If turned off, trading technologies in the diplomacy
+  dialog is not allowed.
+
+``traitdistribution``
+  :strong:`Value (Min, Max)`: Fixed (``FIXED``)
+
+  :strong:`Description`: :term:`AI` trait distribution method. Possible values:
+
+    * ``FIXED``: Fixed
+    * ``EVEN``: Even
+
+``turnblock``
+  :strong:`Value (Min, Max)`: enabled
+
+  :strong:`Description`: Turn-blocking game play mode. If this is turned on, the game turn is not advanced
+  until all players have finished their turn, including disconnected players.
+
+``unitwaittime``
+  :strong:`Value (Min, Max)`: 0 (0, 8639999)
+
+  :strong:`Description`: Minimum time between unit actions over turn change. This setting gives the minimum
+  amount of time in seconds between unit moves and other significant actions (such as building cities) after a
+  turn change occurs. For example, if this setting is set to 20 and a unit moves 5 seconds before the turn
+  change, it will not be able to move or act in the next turn for at least 15 seconds. This value is limited
+  to a maximum value of two-thirds of ``timeout``. ``unitwaittime`` (aka UWT) is a tool to help reduce
+  :term:`RTS` around turn change (:term:`TC`) in Longturn games.
+
+``unitwaittime_extended``
+  :strong:`Value (Min, Max)`: disabled
+
+  :strong:`Description`: ``unitwaittime`` also applies to newly-built and captured/bribed units. If set,
+  newly-built units are subject to ``unitwaittime`` so that the moment the city production was last touched
+  counts as their last *action*. Also, getting captured/bribed counts as action for the victim.
+
+``unitwaittime_style``
+  :strong:`Value (Min, Max)`: empty value/not set
+
+  :strong:`Description`: This setting affects ``unitwaittime`` and effectively unused as it only has one option
+  to set:
+
+    * Activities (``ACTIVITIES``): Units moved less than ``unitwaittime`` seconds from turn change will not
```

Comment:
Option value seems pretty self-explanatory, the short value description seems redundant
```suggestion
    * ``ACTIVITIES``: Units moved less than ``unitwaittime`` seconds from turn change will not
```